test/inline-model-typescript.ts

```diff
@@ -9,7 +9,8 @@
  */
 import {AWS, Client, Entity, Model, Table, dump, print} from './utils/init'
 
-const table = new Table({
+// send any schema because it gets modified
+const table = new Table<any>({
     name: 'InlineModelTypeScriptTestTable',
```

Comment:
Yes -- we should cleanup the any as you say.

Reply:
not sure why its not showing the update here but based on the same change above. The schema now defaults to any if not provided

Reply:
Okay, understand the issue now.